simple_httpfs/simple-httpfs.py

```diff
@@ -13,8 +13,8 @@ def main():
 """)
```

Comment:
All the edits in this file are aggressive edits-on-save from my text editor. Essentially it just strips out unnecessary spaces.